src/scout_apm/core/call_set.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
```

Comment:
Is there a better name than call_set?

`CallCounter` - `SpanCounter` - `SpanCounts` - `TracebackCapturePolicy` - `TracebackNPlusOnePolicy`